libraries/AP_Proximity/AP_Proximity.h

```diff
@@ -191,6 +191,8 @@ class AP_Proximity
     AP_Int8 _raw_log_enable;                            // enable logging raw distances
     AP_Int8 _ign_gnd_enable;                           // true if land detection should be enabled
     AP_Float _filt_freq;                               // cutoff frequency for low pass filter
+    AP_Float _max_m;                                   // Proximity maximum range
+    AP_Float _min_m;                                   // Proximity minimum range
```

Comment:
```suggestion
    AP_Float _max_range_m;                                   // Proximity maximum range
    AP_Float _min_range_m;                                   // Proximity minimum range
```

Reply:
If we're going to rename it let's call them "_distance_min" and "_distance_max".